src/platform/plugins/shared/saved_objects_management/public/services/actions/share_saved_objects_to_space_action.tsx

```diff
@@ -43,6 +43,15 @@ export class ShareToSpaceSavedObjectsManagementAction extends SavedObjectsManage
         !this.actionContext ||
         !!this.actionContext.capabilities.savedObjectsManagement.shareIntoSpace;
       const { namespaceType, hiddenType } = object.meta;
+
+      if (object?.type && object.type.startsWith('ml-')) {
+        const hasMlShareCapabilities =
```

Comment:
this check is only looking at AD capabilities. If I disable AD in my yml file with `xpack.ml.ad.enabled: false` I can no longer share trained models or DFA jobs.